package.json

```diff
@@ -61,12 +62,15 @@
     "deep-shape-equals": "^0.1.2",
     "deepmerge": "^4.2.2",
     "flat": "^5.0.2",
+    "fs-extra": "^10.1.0",
```

Comment:
Do we need this? Node 16+ has much better file copying built-ins that we could use to avoid an extra dep here?

Reply:
If we can replace it easily in https://github.com/primer/primitives/pull/366/files/1113d574244fd445b1bbf6d216a81ee2c5256671#diff-19c3a8039a4d716c096554204131cf6aad8c75aec66e5756c2b0a0e9e5a2d939R12 I am all in.

I think we can now narrow it down to just copying folders with all files inside. This may make it easier, as it does not need to copy files as input. I can also look into to.

Reply:
See: https://github.com/primer/primitives/pull/366#discussion_r992307456